docs/notebooks/pyUmbral Simple API.ipynb

```diff
@@ -93,15 +106,66 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "b'attack at dawn'\n"
+      "b'Proxy Re-encryption is cool!'\n"
      ]
     }
    ],
    "source": [
-    "cleartext = umbral.decrypt(capsule, alice_private_key, ciphertext)\n",
+    "cleartext = pre.decrypt(ciphertext=ciphertext, \n",
+    "\t\t\t\t\t\tcapsule=capsule, \n",
+    "\t\t\t\t\t\tdecrypting_key=alices_private_key)\n",
     "print(cleartext)\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Enter Bob\n",
+    "Apart from generating his keypair, we will also assume that Bob receives a capsule through a side channel (s3, ipfs, Google cloud, etc). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bobs_private_key = keys.UmbralPrivateKey.gen_key()\n",
+    "bobs_public_key = bobs_private_key.get_pubkey()\n",
+    "\n",
+    "bob_capsule = capsule"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Attempt Bob's decryption (fail)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Decryption failed! Bob doesn't has access granted yet.\n"
+     ]
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    fail_decrypted_data = pre.decrypt(ciphertext=ciphertext, \n",
+    "    \t\t\t\t\t\t\t\t  capsule=capsule, \n",
```

Comment:
`\t`?

Reply:
what you don't like tab?

Reply:
Yeah - the tabs are rendered as visible whitespace characters - perhaps replace with spaces for consistency.